src/lpwFuncs.cpp

```diff
@@ -232,37 +234,12 @@ void dormantUntilInterrupt(int interruptPin) {
  * @brief Handles inactivity by transitioning the system to dormant mode.
  */
 void inactivityHandler() {
-  inactivityTime=0;
-  deinitBattery();
-  #ifndef FLIPMOUSE
-    MouseBLE.end();     // turn off Bluetooth
-  #endif
-  #ifdef DEBUG_BATTERY_MANAGEMENT
-    Serial.println("goodbye, going to sleep!");
-  #endif 
-  Serial.flush();
-  Serial.end();
-  displayMessage((char*)"ByeBye");
-  delay(2000);   // time for the user to read the message
-  disable3V3();  // shut down peripherals
-  digitalWrite(LED_BUILTIN,LOW);  // make sure the internal LED is off
-
-  dormantUntilInterrupt(input_map[0]); // enter sleepMode, use Button1 to wakeup!
+  initDormant();
+  dormantUntilInterrupt(input_map, NUMBER_OF_PHYSICAL_BUTTONS); // enter sleepMode, use input_map pins to wakeup!
   //  <--   now sleeping!  
-  
+  cyw43_arch_init();
```

Comment:
not sure if / why this is needed ... shouldn't the cyw43 come up normally after the watchdog reset?